docker-compose.yml

```diff
@@ -0,0 +1,31 @@
+services:
+  db:
+    image: postgres:17-alpine
+    restart: always
+    ports:
+      - "5432:5432"
+    env_file:
+      - .env
+    volumes:
+      - postgres_data:/var/lib/postgresql/data
+
+  app:
+    build: .
+    command: >
+      sh -c "python manage.py wait_for_db &&
+            python manage.py makemigrations &&
+            python manage.py migrate &&
+            python manage.py runserver 0.0.0.0:8000"
```

Comment:
The command chaining multiple `manage.py` commands with `&&` can become hard to manage and debug. Consider using a separate entrypoint script to handle these commands more cleanly.

Reply:
Agree